src/main/java/com/hartwig/platinum/pdl/ConvertFromGCSPaths.java

```diff
@@ -28,34 +32,34 @@ public static List<TumorNormalPair> apply(final List<SampleConfiguration> config
             if (!sample.tumors().isEmpty()) {
                 for (RawDataConfiguration tumor : sample.tumors()) {
                     if (tumor.bam().isPresent()) {
-                        pairs.add(ImmutableTumorNormalPair.builder()
-                                .reference(sample.normal().map(n -> ImmutableSample.builder().name(n.name()).bam(n.bam()).build()))
-                                .tumor(ImmutableSample.builder()
+                        pairs.add(ImmutablePipelineInput.builder()
+                                .reference(sample.normal().map(n -> ImmutableSampleInput.builder().name(n.name()).bam(n.bam()).build()))
+                                .tumor(ImmutableSampleInput.builder()
                                         .name(tumor.name())
                                         .bam(tumor.bam())
                                         .primaryTumorDoids(sample.primaryTumorDoids())
                                         .build())
-                                .tumorIndex(indexTumors ? Optional.of(tumorIndexString(tumorIndex)) : Optional.empty())
-                                .name(indexTumors ? sample.name() + "-" + tumorIndexString(tumorIndex) : sample.name())
+                                .setName(indexTumors ? sample.name() + "-" + tumorIndexString(tumorIndex) : sample.name())
```

Comment:
Let's remove the `tumorIndexString` method. In the new implementation it doesn't add a lot.

Reply:
I meant we could just inline the `"t"`